spec/system/account_modal_system_spec.rb

```diff
@@ -64,30 +67,29 @@
     expect(page).to have_link(id: 'edit_account_button')
     click_link(id: 'edit_account_button')
 
+    sleep(2)
```

Comment:
avoid using sleep in specs !
usually it's better to add another expect page to have ...

Reply:
I deleted the sleep statement everywhere and replaced it with expect.